bin/utils/utils.js

```
@@ -1,7 +1,74 @@
+
+const axios = require("axios");
+const chalk = require("chalk");
+
 class FlightTrackerCLI {
-  constructor() {}
+  constructor() {
+    this.axiosInstance = axios.create({
+      baseURL: 'https://api.aviationstack.com/v1',
+      timeout: 5000,
+      headers: {
+        'Content-Type': 'application/json',
+      }
+    });
+  }
+
+  async findIATACode(airportName) {
+    try {
+      const response = await this.axiosInstance.get('/airports', {
+        params: {
+          city: airportName,
+          // access_key: 'YOUR_API_KEY', // option to pass key manually
```

We can add the api key as a class property. If this property is null, we should throw an error telling the user no API key has been found and that they need to run the appropriate commands.

---

Ahh yeah that is really good.